config/packages/dev/swiftmailer.yaml

```diff
@@ -1,5 +0,0 @@
-# See https://symfony.com/doc/current/email/dev_environment.html
-swiftmailer:
-    # send all emails to a specific address
-    #delivery_addresses: ['me@example.com']
-    port: 1025
```

Comment:
You removed the three swiftmailer configs, I'd expect new (framework) config settings for these environments that reflect the settings that where configured before.